### ISSUE TITLE
ContentMine Fellowships

### ISSUE THREAD

**Issue by jcmolloy** [ContentMine/site#344]
ContentMine have decided to offer small amounts of funding to five early career researchers to be ContentMine fellows.

@jcmolloy will complete the following by 1 April
- [x] Draft details for feedback from team [[DONE](https://docs.google.com/document/d/1KbLD5Ax24QSL_IS4GLZgfVgQMEEEx49CFwBC9JTGAA4/edit#heading=h.pmrlrnyx5e6o)]
- [x] Draft blog post [[DONE](http://contentmine.org/wp-admin/post.php?post=1685&action=edit)]
- [x] Draft mailing list email [[DONE](https://docs.google.com/document/d/1_jyQrqZZMHlDNPmf4dtEdOC49V_xETjy_BoBjzGz34k/edit#)]
- [x] Make social media graphic [[DONE](https://drive.google.com/open?id=0B9V3tIqIu0UZeGpBTlU0ZGh6U0U)]
- [x] Submit Shuttleworth pitch [[DRAFT](https://docs.google.com/document/d/159IgHnuA2OY6dcnRQYIQ-GNdz6S6AUlRbuEnvKtHLPw/edit#)]

@GrahamSteel will then perform the following tasks and advertise from 8 May
- [x] Post blog on contentmine.org and make a Discourse category for Fellowships containing the blog anmouncement, add a reply offering to answer any questions people have (don't need to do them all yourself - delegate as necessary!)
- [x] Tweet frequently from CM and mention relevant orgs/people
- [x] Send to mailing lists e.g. Mozilla Science, OKFN, OpenCon
- [x] Ask team to send out through their networks and remind at weekly meetings
- [x] Respond to applicants with confirmation email
- [x] Make a folder for each applicant in the [Fellowships GDrive folder](https://drive.google.com/drive/?usp=kix-move#my-drive?action=locate&id=1KbLD5Ax24QSL_IS4GLZgfVgQMEEEx49CFwBC9JTGAA4&parent=0B9V3tIqIu0UZUUVHaUZtZWV2aFU) and copy in all relevant files

Applications close on 3 June, @jcmolloy and @viviennelm will coordinate shortlisting and interviews
- [x] Arrange time for shortlisting
- [x] Inform applicants of shortlist results
- [x] Send shortlist instructions to install ContentMine and perform a task (TBD by @tarrow @petermr @chreman @skasberger)
- [x] Arrange virtual interviews week beginning 16 June
- [x] Respond to all applicants

New issue will be started for implementation of the fellowships

**Comment by jcmolloy** [comment]
Assigning to @GrahamSteel for advertising once pitch is approved

**Comment by jcmolloy** [comment]
Thanks for all your hard work on this Graham! I am reassigning myself for the interview phase.

**Comment by jcmolloy** [comment]
Completed and next phase documented in #361
